mithril-common/src/cardano_transactions_preloader.rs

```diff
@@ -14,6 +15,32 @@ use crate::signable_builder::TransactionsImporter;
 use crate::signed_entity_type_lock::SignedEntityTypeLock;
 use crate::StdResult;
 
+/// CardanoTransactionsPreloaderActivationState
```

Comment:
The name is not very clear, maybe this could be better as it expresses what it is doing:
```suggestion
/// CardanoTransactionsPreloaderChecker
```

Also a sentence is welcome to explain the purpose of the trait, just its name is not enough.